DualView/app/src/main/java/com/microsoft/device/display/samples/dualview/MainActivity.kt

```diff
@@ -0,0 +1,67 @@
+/*
+ *
+ *  * Copyright (c) Microsoft Corporation. All rights reserved.
+ *  * Licensed under the MIT License.
+ *  *
+ *
+ */
+
+package com.microsoft.device.display.samples.dualview
+
+import android.content.res.Configuration
+import android.os.Bundle
+import android.os.Handler
+import android.os.Looper
+import androidx.appcompat.app.AppCompatActivity
+import androidx.compose.ui.platform.setContent
+import androidx.lifecycle.ViewModelProvider
+import androidx.window.WindowManager
+import com.microsoft.device.display.samples.dualview.models.AppStateViewModel
+import com.microsoft.device.display.samples.dualview.ui.theme.DualViewComposeSampleTheme
+import java.util.concurrent.Executor
+
+class MainActivity : AppCompatActivity() {
+    private lateinit var windowManager: WindowManager
+    private lateinit var appStateViewModel: AppStateViewModel
+
+    private val handler = Handler(Looper.getMainLooper())
+    private val mainThreadExecutor = Executor { r: Runnable -> handler.post(r) }
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        windowManager = WindowManager(this)
+        appStateViewModel = ViewModelProvider(this).get(AppStateViewModel::class.java)
+        appStateViewModel.setSelectionLiveData(-1)
```

Comment:
please, don't use magic numbers, better to have it in a constant with a meaningful name, so we know what that number represents.